### ISSUE TITLE
Terminal accessibility menu keybinding not read with screen reader attached 

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#15785]
Testing -171862

With editor.accessibilitySupport enabled and Windows Narrator running, when I click into the terminal, Narrator reads the terminal number, type (pwsh), and keybindings to navigate within the terminal history, but does not read the keybinding for Show Terminal Accessibility Help. Shortcuts are unchanged from defaults:

<img width="1184" alt="image" src="https://user-images.githubusercontent.com/30305945/214355855-e82d33b7-2d0e-4639-be84-d412633fa386.png">
